src/main.scala

```diff
@@ -5,7 +5,7 @@ import scala.collection.mutable.ListBuffer
  */
 object main extends App {
   
-  val network = List(
+  var network = List(
```

Comment:
Oops, this shouldn't be here.  Need to make this a value again.
Had changed it only for a copy paste into the REPL for testing purposes.